server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -251,9 +251,15 @@ public final class IndexSettings {
         Property.IndexScope
     );
     public static final TimeValue DEFAULT_REFRESH_INTERVAL = new TimeValue(1, TimeUnit.SECONDS);
+    public static final Setting<TimeValue> NODE_DEFAULT_REFRESH_INTERVAL_SETTING = Setting.timeSetting(
+        "node._internal.default_refresh_interval",
```

Comment:
I see some other internal settings like [this one](https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/indices/recovery/RecoverySettings.java#L288) named `indices.recovery.internal_action_retry_timeout`, and I wonder if for the sake of consistency, this might be better renamed to `node.internal_default_refresh_interval` or something like that. Somehow I also think I have not yet seen a setting start with `_`.

Reply:
I spoke to @henningandersen about this yesterday and `_internal` was the preferred naming to discourage use. It's also not clear to me the meaning behind `internal` in `internal_action_timeout`. We actually [publicize](https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-settings.html) this setting in CCR. 

Regardless we can change or remove this setting since it is not a cluster setting and not documented. I'm merging atm to unblock downstream testers.